source/TR-181/middle_layer_src/wanmgr_rdkbus_utils.c

```diff
@@ -66,6 +66,94 @@
 extern char g_Subsystem[32];
 extern ANSC_HANDLE bus_handle;
 
+ANSC_STATUS WanMgr_SetConnectivityCheckTypeToPSM(DML_VIRTUAL_IFACE* pVirtIf, CONNECTIVITY_CHECK_TYPE type)
+{
+    int result = ANSC_STATUS_SUCCESS;
+    int retPsmSet = CCSP_SUCCESS;
+    char param_name[BUFLEN_256] = {0};
+    char param_value[BUFLEN_256] = {0};
+
+    /* Update the wan policy information in PSM */
+    memset(param_value, 0, sizeof(param_value));
```

Comment:
memset required?

Reply:
kind of improvement, I hope assignment would work well, i will changes it.